### PR TITLE
DOC: Simplify bug report template

### PR DESCRIPTION
**Pull request by lassoan**
Removed expected behavior section, as it is normally described in the "Steps to reproduce" section (what you did + what you expected to happen + what happened instead).

### CODE REVIEW

**Review by jcfr — APPROVED** [review]

**Review by jcfr — COMMENTED** [review]
Reviewing [existing reports](https://github.com/Slicer/Slicer/issues?q=is%3Aissue+is%3Aopen+%23%23+Expected+behavior) is also informative.

Since the step to reproduce may be lengthy, having an explicit section ensure we have a short and concise summary.

For some examples:
* https://github.com/Slicer/Slicer/issues/7206
* https://github.com/Slicer/Slicer/issues/7124

For this reason, I think we could keep the `Expected behavior`

**Comment by lassoan** [comment]
Often I need to describe "Actual behavior" and "Expected behavior" for several steps within "How to reproduce" section. For example:

```
1. Do A
2. Do B => Slicer does Y - OK
3. Do C => Slicer does Z - WRONG -> Slicer should do W instead
4. Do D => Slicer does Z - OK
5. Do B => Slicer does Y - WRONG -> Slicer should do X instead
```

I find it very hard to use the existing template for this, because:
- I am forced to describe only a single "expected behavior" (cannot describe multiple places where Slicer behaved in some unexpected ways),
- I am forced to describe the actual behavior somewhere else (in a different section), and
- I cannot link the expected behavior description to specific reproduction steps.

"Actual behavior" section is obviously missed. We could add a dedicated "Actual behavior" section (see for example in the [ITK issue template](https://github.com/InsightSoftwareConsortium/ITK/issues/new?assignees=&labels=type%3ABug&projects=&template=bug_report.md)) or rename the section to "Actual and expected behavior", but it would not address the other limitation that I described (report "what you did, what you expected to happen, what happened instead" information close together, allow reporting multiple of these within one issue).

**Comment by jamesobutler** [comment]
> ```
> 1. Do A
> 2. Do B => Slicer does Y - OK
> 3. Do C => Slicer does Z - WRONG -> Slicer should do W instead
> 4. Do D => Slicer does Z - OK
> 5. Do B => Slicer does Y - WRONG -> Slicer should do X instead
> ```

That is good informative example. I think it would be good to include that in places of explaining in pure paragraph text. People go a bit blind when viewing a text block and may not read it. The numbered list stands out and is quick to serve as an example even if the person doesn’t read any other text.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -13,11 +13,7 @@ assignees: ''
...
-
-## Expected behavior
-
-<!-- Describe what you expected to happen. -->
+<!-- Generally, Slicer developers can only fix those issues that they can reproduce on their computers. To achieve this, please describe in this section what you did, what you expected to happen, and what happened instead. Attach a few screenshots if possible. Use one of the Slicer sample data sets (in Sample Data module) as inputs. If you cannot reproduce the problem with sample data sets then you can upload your data somewhere and provide a download link here (make sure to remove all patient information from the data before sharing). If the problem occurs by using the application from custom code, create a short self-contained example as described at http://sscce.org -->
```
Comment (jcfr) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]:
```suggestion
<!--
    Generally, Slicer developers can only fix those issues that they can reproduce on their computers.
    
    To achieve this, please:
    * Describe in this section what you did, what you expected to happen, and what happened instead.
    * Attach a few screenshots if possible.
    * Use one of the Slicer sample data sets (in Sample Data module) as inputs. If you cannot reproduce the problem with sample data sets then you can upload your data somewhere and provide a download link here (make sure to remove all patient information from the data before sharing).
  
  If the problem occurs by using the application from custom code, create a short self-contained example as described at http://sscce.org
  
  To describe "Actual behavior" and "Expected behavior" you would use the following format:
  1. Do A
  2. Do B => Slicer does Y - OK
  3. Do C => Slicer does Z - WRONG -> Slicer should do W instead
  4. Do D => Slicer does Z - OK
  5. Do B => Slicer does Y - WRONG -> Slicer should do X instead
-->
```

**Comment by jcfr** [comment]
@lassoan @jamesobutler Thanks for the very informative follow-up, I suggested a change to help with visual parsing and further help guide the reporting process.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -13,11 +13,23 @@ assignees: ''
...
-<!-- Describe what you expected to happen. -->
+<!--
+    Generally, Slicer developers can only fix those issues that they can reproduce on their computers.
+    
+    To achieve this, please:
+    * Describe in this section what you did, what you expected to happen, and what happened instead.
+    * Attach a few screenshots if possible.
+    * Use one of the Slicer sample data sets (in Sample Data module) as inputs. If you cannot reproduce the problem with sample data sets then you can upload your data somewhere and provide a download link here (make sure to remove all patient information from the data before sharing).
+  
+    If the problem occurs by using the application from custom code, create a short self-contained example as described at http://sscce.org
```
Comment (lassoan) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]:
```suggestion
    If the problem cannot be reproduced by using the graphical user interface but only by running custom Python or C++ code then please create a short self-contained example as described at http://sscce.org
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -13,11 +13,23 @@ assignees: ''
...
+  
+    To describe "Actual behavior" and "Expected behavior" you may use the following format:
+    1. Do A
+    2. Do B => Slicer does Y - OK
+    3. Do C => Slicer does Z - WRONG -> Slicer should do W instead
```
Comment (lassoan) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]:
```suggestion
    3. Do C => Slicer does Z => ERROR: Slicer should do W instead
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -13,11 +13,23 @@ assignees: ''
...
+    4. Do D => Slicer does Z - OK
+    5. Do B => Slicer does Y - WRONG -> Slicer should do X instead
```
Comment (lassoan) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]:
```suggestion
    5. Do B => Slicer does Y => ERROR: Slicer should do X instead
```

**Comment by lassoan** [comment]
Thanks for the suggestions and updates. I've made a few tweaks, please have a look and approve if it looks OK to you.

**Review by jcfr — APPROVED** [review]
<s>Make sure to select "Squash & Merge" setting the title to `DOC: Simplify bug report template`</s>

Commits have been squashed & pull request updated :white_check_mark:

**Comment by cpinter** [comment]
Looks great to me too. Thank you

**Comment by lassoan** [comment]
Thank you @jcfr merging and everyone for the review&comments.

**Comment by jcfr** [comment]
@lassoan Can you also review and approve this follow up pull request :pray: 
* https://github.com/Slicer/Slicer/pull/7334

**Pull request merged**